README.md

```diff
@@ -42,6 +42,31 @@ const sdk = await new AppExtensionsSDK({ identifier: '123abc' })
   .initialize({ size: { height: 500 } });
 ```
 
+### Without module bundler
+
+Initialization without a module bundler is possible by adding the following script to your HTML page from jsDelivr CDN:
+
+```HTML
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/@pipedrive/app-extensions-sdk@0.8.0/dist/index.umd.js"></script>
+</head>
+```
+
+NB: Pay attention to the package version in the URL - `app-extensions-sdk@0.8.0`. While you can also use version range,
+ensure you do not omit the version completely in production. This is to avoid any issues.
+
+After this, the global `AppExtensionsSDK` will be available. Initialization can be then done the same way below in the HTML body:
```

Comment:
Can you change it to `AppExtensionsSDK` so it matches the naming format we use elsewhere in the readme?

Reply:
Done ✅